src/03-deals/quotations.apib

```diff
@@ -156,4 +156,41 @@ Download a quotation in a specific format.
     + Attributes (object)
         + data (object)
             + type: `quotation` (string)
-            + id: `e4bf74ca-e900-471f-84b1-276e5d3afae4` (string)
\ No newline at end of file
+            + id: `e4bf74ca-e900-471f-84b1-276e5d3afae4` (string)
+
+### quotations.update [POST /quotations.update]
+
++ Request (application/json)
+
+    + Attributes (object)
+        + quotation_id: `5b16f6ee-e302-0079-901b-50c26c4a55b1` (string, required)
```

Comment:
Didn't notice it in the first review, but the subject's ID is usually passed as `id`.
```suggestion
        + id: `5b16f6ee-e302-0079-901b-50c26c4a55b1` (string, required)
```